rest/src/main/java/com/redhat/ipaas/rest/v1/handler/connection/ConnectorHandler.java

```diff
@@ -45,4 +56,107 @@ public ConnectorActionHandler getActions(@PathParam("id") String connectorId) {
         return new ConnectorActionHandler(getDataManager(), connectorId);
     }
 
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{id}/verify")
+    public ValueResult<String> verify(@PathParam("id") String connectorId, Properties configuredProperties) throws IOException {
```

Comment:
I really would delegate that to an extra module in #122 (hey, already was prepared there already ;-). This could be easily switch off e.g. with no-op implementation for UI development. But see #122 for details.